examples/run_torch_onnx.py

```diff
@@ -0,0 +1,163 @@
+# Copyright (C) 2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+import argparse
+import importlib
+import sys
+from pathlib import Path
+
+import cv2
+import numpy as np
+import openvino as ov
+import timm
+import torch
+
+from openvino_xai import Task, insert_xai
+from openvino_xai.common.utils import logger, softmax
+from openvino_xai.explainer.visualizer import colormap, overlay
+
+
+def get_argument_parser():
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--model_name", default="resnet18.a1_in1k", type=str)
+    parser.add_argument("--image_path", default="tests/assets/cheetah_person.jpg", type=str)
+    parser.add_argument("--output_dir", default=".data/example", type=str)
+    return parser
+
+
+def main(argv: list[str]):
+    parser = get_argument_parser()
+    args = parser.parse_args(argv)
+
+    # Load Torch model from timm
+    try:
+        model = timm.create_model(args.model_name, in_chans=3, pretrained=True)
+        logger.info(f"Model config: {model.default_cfg}")
+        logger.info(f"Model layers: {model}")
+    except Exception as e:
```

Comment:
Can you move this block and everything below in the separate function "insert_xai_to_torch_export_to_ir" and copy the insert part and create separate "insert_xai_to_torch_export_to onnx". Please add doc strings as well. It will create similarity between our "run_" files.
If you think, it's not needed, then can you just add a docstring to main?

Reply:
Done in https://github.com/openvinotoolkit/openvino_xai/pull/66/commits/202b049f40609892d63637a8824730aaf47bc6ff